app/dashboard/__init__.py

```diff
@@ -2,9 +2,10 @@
 # Author: Matt Hart <matthew.hart@linaro.org>
 # Author: Milo Casagrande <milo.casagrande@linaro.org>
 #
-# Copyright (C) Collabora Limited 2018
+# Copyright (C) Collabora Limited 2018,2019
```

Comment:
Well, it looks like the patch came from @oussemadaoud.  So while the open source license allows you to reuse the code, you don't get copyright if you simply cherry-pick that change and don't add any other substantial contributions to it.

It's not a big issue and this a bit of a grey area, but it would simplify the PR to leave the copyright headers unchanged (and imo they shouldn't be changed in this case).